src/angular/translations/translations.config.js

```diff
@@ -33,6 +33,7 @@ servicesAdvisorApp.config(['$translateProvider', function ($translateProvider) {
     'SHELTER': 'SHELTER',
     'WASH': 'WASH',
     "REFERRAL_METHOD": "Referral Method",
+    "CREATE_REVIEW": "Create Review",
```

Comment:
Might as well google translate that to arabic too, and then keep a list somewhere of pending translations.